homeassistant/components/plugwise/translations/select.en.json

```diff
@@ -1,11 +1,17 @@
 {
     "state": {
-        "plugwise__regulation_mode": {
-            "bleeding_cold": "Bleeding cold",
-            "bleeding_hot": "Bleeding hot",
-            "cooling": "Cooling",
-            "heating": "Heating",
-            "off": "Off"
-        }
+      "plugwise__regulation_mode": {
+        "bleeding_cold": "Bleeding cold",
+        "bleeding_hot": "Bleeding hot",
+        "cooling": "Cooling",
+        "heating": "Heating",
+        "off": "Off"
+      },
+      "plugwise__dhw_mode": {
+        "off": "Off",
+        "auto": "Auto",
+        "boost": "Boost",
+        "comfort": "Comfort"
+      }
     }
-}
\ No newline at end of file
+  }
```

Comment:
Bad formatting.

Reply:
Yes I noticed, however, formatting isn't enforced on translations and are overwritten each nightly download anyways. So, I figured it was safe to ignore.